liferay-tango/liferay-portal
# LPS-151362 Minor fix in Experiences dropdown to show a warning message when segmentation is disabled

Motivation
=======

Minor fix for when to show the warning message in the experiences dropdown.

[Link to story or ticket](https://issues.liferay.com/browse/LPS-151362)

Proposed Solution
========
Instead of using the _groupCompanyId_ to check if the message should be shown or not, the _companyId_ must be used

Steps to Verify:
----------------

1. Disable Segmentation: Go to Instance Settings > Segments > Segments Service.
2. Go to main page and click the edit icon
3. Deploy the Experiences dropdown
4. The corresponding message should be visible:

- If segmentation is enabled: Warning Changes to experiences are applied immediately
- If segmentation is disabled: Experiences can not be displayed because segmentation is disabled. To enable Segmentation go to Instance Settings > Segments > Segments Service.

Change summary:
---------------

- LPS-151362 Warning message in experiences when segmentation is disabled: Minor fix when retrieving companyId
- LPS-151362 Lang key rename
